content/blog/2022-02-28-february-22-community-gems.md

```diff
@@ -0,0 +1,155 @@
+
+This is another fantastic question from @jotsif!
+
+No, we deliberately terminate the instance to avoid unexpected costs. Stopped
+but unterminated instances
+[can still cost the same as running ones](https://aws.amazon.com/premiumsupport/knowledge-center/ec2-billing-terminated/).
+It's best to let the CML runner terminate and create new instances, running
+`dvc pull` to restore your data each time.
+
+However, if you're trying to preserve data (e.g. cache dependencies to speed up
+experimentation time) on an AWS EC2 instance, you could
+[connect persistent AWS S3 remote storage](https://aws.amazon.com/premiumsupport/knowledge-center/s3-transfer-data-bucket-instance/).
+
+### [What's the difference between DVC Studio free and enterprise versions?](https://discord.com/channels/485586884165107732/841856466897469441/933324508570472497)
+
+Thanks for asking @Abdi!
+
+You can find more info about the different
+[DVC Studio tiers here](https://studio.iterative.ai/#pricing).
+
+The _Free_ tier has all the features most individual users need, like connecting
+to ML repositories, creating views, submitting experiments, and generating
+plots. The _Teams_ tier allows you to create large teams for better
+collaboration and sharing of views and settings with everyone. The _Enterprise_
+tier is more for needs around compliance, dedicated support, and on-premise
+installation.
+
+If you are trying to decide which plan to select, please email us at
+`support@iterative.ai` and we'll help you figure it out based on your needs.
+
+### [How can I use one `dvc.yaml` file with multiple pipeline folders with different `params.yaml` files?](https://discord.com/channels/485586884165107732/485596304961962003/939099847288578079)
+
+@louisv, thanks for this question!
+
+It seems like you're looking for the parametrization functionality. You can
+learn more about how it works
+[in this doc](https://dvc.org/doc/user-guide/project-structure/pipelines-files#templating),
+but here's a an example of what that might look like in the `dvc.yaml`.
+
+```yaml
+stages:
+  cleanups:
+    foreach: # List of simple values
+      - raw1
+      - labels1
+      - raw2
+    do:
+      cmd: clean.py "${item}"
+      outs:
+        - ${item}.cln
+```
+
+### [Is it possible to change the x-label in DVC Studio?](https://discord.com/channels/485586884165107732/841856466897469441/938857004187943003)
+
+A great question about Studio from @PythonF!
+
+You can set custom properties for your plot in your `dvc.yaml` like this:
+
+```yaml
+plots:
+  - plots_no_cache.csv:
+      cache: false
+      x: r
+```
+
+You can also use `dvc plots modify` to change the x-label or y-label for your
+plots using commands similar to the following.
+
+```dvc
+$ dvc plots modify plots_no_cache.csv -x r -y q
+```
+
+---
+
+https://media.giphy.com/media/h5Ct5uxV5RfwY/giphy.gif
+
+At our March Office Hours Meetup we will be about how you can create, run, and
+benchmark DVC pipelines with ZnTrack!
```

Comment:
Can we add a link to https://github.com/zincware/ZnTrack to help promote this awesome project?